Gemfile.lock

```diff
@@ -1,13 +1,13 @@
 PATH
   remote: .
   specs:
-    ssf (0.0.12)
-      google-protobuf (= 3.3.0)
+    ssf (0.0.13)
+      google-protobuf (~> 3.3)
 
 GEM
   remote: https://rubygems.org/
   specs:
-    google-protobuf (3.3.0)
+    google-protobuf (3.3.0-universal-darwin)
```

Comment:
Is this right? Our Jenkins builds are on Linux, not OS X.

Reply:
Ah, probably not, good catch.

Don't think it matters in practice--this file should only be used in this build, which passes, and anything that depends on the gem would just look at the gemspec--but wasn't a deliberate change & probably should undo it anyway.